client/dogs/FormPage.js

```diff
@@ -363,8 +364,8 @@ export default class IndexPage extends React.Component {
         color: result['priColor'] || '',
         breed: result['priBreed'] || '',
         sex: result['sex'] || '',
-        rabiesIssued: result['vacDate'] || '',
-        rabiesExpire: result['vacExp'] || '',
+        rabiesIssued: Moment(result['vacDate']).format('L') || '',
+        rabiesExpire: Moment(result['vacExp']).format('L') || '',
```

Comment:
Totally agree, yup that's how the lower case `l` seems to work. I only went with uppercase because that was what was used in the placeholder attr and was the format referenced in issue #26. I've updated the format and placeholder in the latest commit. Thanks!